be/src/olap/tablet.h

```diff
@@ -328,7 +328,11 @@ class Tablet : public BaseTablet {
 
     static void remove_unused_remote_files();
 
-    std::shared_mutex& get_remote_files_lock() { return _remote_files_lock; }
+    // If a rowset is to be written to remote filesystem, MUST add it to `pending_remote_rowsets` before uploading,
+    // and then erase it from `pending_remote_rowsets` after it has been insert to the Tablet.
+    // `remove_unused_remote_files` MUST NOT delete files of these pending rowsets.
+    static void add_pending_remote_rowset(std::string rowset_id);
```

Comment:
Why static？s_pending_remote_rowsets will be too large, and take too long time when searching key.